specs.md

```diff
@@ -68,13 +68,13 @@ FSM-18 | Brake actuation reporting | - The FSM must report the brake actuation O
 ID | Title | Description | Associated Competition Rule(s)
 --- | --- | --- | ---
 FSM-17 | Default state | The FSM state machine must begin in the AIR-Open state by default.
-FSM-12 | Exiting the AIR-Open state | The FSM must enter the AIR-Closed state when the BMS closes the AIR+ and AIR-.
+FSM-12 | Exiting the AIR-Open state | The FSM must enter the AIR-Closed state when the BMS closes the AIR+. 
```

Comment:
@kirons96 Should I also add "or when the BMS detects AIR+ and AIR- are closed" or does this suffice

Reply:
AIR+ alone should be fine as AIR- is less meaningful (i.e. during precharge)

Reply:
@kirons96 Actually before I merge this in could you explain why AIR- is less meaningful

Reply:
AIR- closes during precharge
AIR+ is open during that time
we don't want to be in the "Air-Closed state" during precharge, only after it